src/builtins/BuiltinTemporal.cpp

```diff
@@ -232,7 +232,7 @@ static Value builtinTemporalPlainTimeConstructor(ExecutionState& state, Value th
 
 static Value builtinTemporalPlainTimeFrom(ExecutionState& state, Value thisValue, size_t argc, Value* argv, Optional<Object*> newTarget)
 {
-    Value options = Temporal::getOptionsObject(state, argc > 1 ? argv[1] : Value());
+    Value options = Temporal::getOptionsObject(state, argc > 1 ? &argv[1] : nullptr);
```

Comment:
Why does it change to `nullptr`?
For undefined case, return just `Value()`.
Or if you want to return `null` primitive JS type, return with `Value(Value::Null)`

Reply:
The aim is reducing memory consumption. `Value(Value::Null)` requires no allocation, so that is also a good option. Shall we use that?

Reply:
Yes, please change it.
IMHO the cost of `Value()` or `Value(Value::Null)` is insignificant.
And assigning `nullptr` to `Value` might be recognized as a Value that points to any valid pointer value.